packages/desktop-app/src/main.ts

```diff
@@ -366,19 +377,30 @@ const createMainWindow = () => {
   })
 
   bridge.on(showNotification, (message: {}) => {
-    notifier.notify(
-      {
-        ...message,
+    if (process.platform === 'win32') {
+      notifier.notify(
+        {
+          ...message,
+          icon: icons.NOTIFICATION_ICON_PATH,
+          appID: 'salad-technologies-desktop-app',
+        },
+        (err) => {
+          if (err) {
+            console.warn('Notification error')
+            console.warn(err)
+          }
+        },
+      )
+    } else {
```

Comment:
Let's check to ensure notifications are supported:

```suggestion
    } else if (Notification.isSupported()) {
```

Docs: https://www.electronjs.org/docs/api/notification#notificationissupported